app/views/hub/efile_submissions/show.html.erb

```diff
@@ -1,22 +1,28 @@
+<% content_for :back_to, "efile" %>
 <% content_for :card do %>
-  <div class="slab slab--not-padded spacing-above-25">
-    <h1 class="h2 spacing-below-5">
-      Submission <%= @efile_submission.id %> <span class="small">(<%= @efile_submission.current_state %>)</span>
-    </h1>
-    <p class="spacing-above-5 spacing-below-5">Client: <%= link_to @efile_submission.tax_return.client.legal_name, hub_clients_path(id: @efile_submission.tax_return.client.id) %></p>
+  <div class="slab slab--not-padded spacing-above-25 submission">
+    <div style="display: flex;">
+      <h1 class="h2 spacing-below-5">
+        <%= link_to @tax_return.client.legal_name, hub_clients_path(id: @tax_return.client.id) %> (<%= @efile_submission.current_state %>)
```

Comment:
I think it's confusing to have the client be a link within a H1/H2, but if you think it makes sense, I am fine with it. (Not a blocker since this is just me expressing confusion.)

Reply:
i agree it's weird and that there is no particular reason it needs to be done this way. however, it was a change requested by design and i already used my energy/goodwill negotiating for other changes that were out of scope on this page.